app/controllers/conversations_controller.rb

```diff
@@ -0,0 +1,23 @@
+class ConversationsController < ApplicationController
+
+  def show
+    @conversation = Conversation.find_by(id: params[:conversation_id])
+  end
+
+  def new
+    @topic = Topic.find(params[:topic_id])
+    @conversation = @topic.conversations.new
+  end
+
+  def create
+    @topic = Topic.find(params[:topic_id])
+    @conversation = @topic.conversations.new(name: params[:conversation][:name])
+    @conversation.user_id = current_user.id
+    if @conversation.save
```

Comment:
This is a good instance of why model validations are important. Right now, since you don't have any validations, a conversation can be saved without a name, exist in your database but not appear on the page.